spec.bs

```diff
@@ -358,7 +358,7 @@ This specification adds a new property to the {{RequestInit}} dictionary:
 
 <pre class=idl>
 partial dictionary RequestInit {
-  PrivateStateToken privateStateToken;
+  PrivateToken privateToken;
```

Comment:
This means that web developers will now need to use `privateToken` in their fetch code. That's a pretty major web-exposed change. I don't think you've changed enough of the spec to account for this rename. There's at least one more example that you still have to change, but also what about the `Sec-Private-State-Token` header? It's inconsistent with this name now.

I would encourage more discussion on whether "PrivateToken" should be the developer-facing term when the spec name is Private State Tokens.

Reply:
Your arguments make sense. I agree on renaming headers as well. 

I agree on spec name as well. Basically two new things are being introduced. (1) A PrivateToken API and (2) A specific implementation of this API (PST). I think having this separation in mind would help with extensibility and backward compatibility. 

This was a decision we discussed when renaming Trust Tokens as Private State Tokens. PrivateStateToken as a dictionary is also new to developers (TrustToken is used during OT). 

Checking latest relevant Privacy Pass spec might help with the context https://www.ietf.org/archive/id/draft-ietf-privacypass-auth-scheme-08.html#section-2.1

I would love to hear what others think and more discussion on this is welcome.

Reply:
Ultimately you're the spec editor and the naming is up to you 😃 , but what I'd like to see is a brief document that describes the new naming strategy and how it will affect this API (i.e. which pieces would be named how), because this seems like a bit of an ad-hoc change that might leave out some details.

Reply:
I thought more about this. 

1. I plan to rename the PrivateStateToken dictionary to PrivateToken. Let the token type specified in the type field. This is in line with our discussions during the renaming decision.

2. Keep HTTP headers as they are (PST specific). This means HTTP headers will not have type and version data and they will have PST in their name. This is fine since only Private State Tokens are supported for now. We need to get back to this when we add support for new type of tokens. I think that would be a good time to consider adopting latest Privacy Pass specs. This is a backward compatibility issue, however for issuers only. Note that number of issuers are expected to be significantly less that the number of pages adopting PST.

Reply:
Ok, as discussed, can you please update the spec to explain what the goal of the PrivateToken abstraction is?

There's also the second example which still uses the PrivateStateToken name.

Reply:
Updated the Goals section. Might add more details in later sections. I will read it thoroughly from beginning and might create more PRs. Thanks.